_infra/helm/secure-message-v2/values.yaml

```diff
@@ -13,7 +13,7 @@ database:
   secrets:
     usernameKey: username
     passwordKey: password
-    nameKey: securemessages
+    nameKey: db-name
```

Comment:
Successfully tested the deployment into a Cloud SQL environment and config/performance/secure-message-v2 overrides this as expected 👍